test/persist_transaction.js

```diff
@@ -329,7 +329,7 @@ describe('persistor transaction checks', function () {
         expect(Object.keys(tx.dirtyObjects).length).to.equal(0);
     });
 
-    it('when an array of field used without providing the table name..', function () {
+    it('when an array of fields used without providing the table name..', function () {
```

Comment:
while we're correcting some grammar here, do you think this should be "when an array of fields _are_ used without..."? Same for below